packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -153,45 +153,46 @@ export interface DropTargetDelegate {
 
 export interface DroppableCollectionProps {
   /**
-   * A function returning the drop operation to be performed when items matching the given types are dropped
-   * on the drop target.
+   * The drag types that the droppable collection accepts. If the collection accepts directories, include 'directory' in your array of allowed types.
+   * @default 'all'
```

Comment:
Noticed this when pulling in the docs into my branch, should this still mention the Directory drag type symbol?
```
The drag types that the droppable collection accepts. If directories are accepted, include the DIRECTORY_DRAG_TYPE from @react-aria/dnd in the array of allowed types.
```

Reply:
Followup: the directory drop examples will need to be updated to use the DIRECTORY_DRAG_TYPE, right now they don't actually accept the directory drop